### ISSUE TITLE
Issue is not related

### ISSUE THREAD

**Issue by randomile** [mamirov/jirareporter#17]
Hello.
We have got a trouble.
If we are choosing "Get issue from: Jira Reporter" it is working. But if we are choosing "Get issue from: VCS comment", it isn't working. We have a message "Issue is not related" in log.
And earlier it was working, but not now. 

What wrong? How we can debug it?

**Comment by mamirov** [comment]
Hi!
Can you try configure your issue tracker connection?

Info from Wiki:
> VCS Comment - issue id is taken from vcs commit, for that it’s needed to set up integration of TeamCity, Administration -> Issue Tracker -> Create new connection.

**Comment by randomile** [comment]
Можно на русском ? )
Спасибо за оперативный ответ.

Да, конечно, интеграция настроена. Необходимые ключи проектов указаны. При нажатии на кнопку Test Connection результат успешный.

С уважением, Виталий

**Comment by mamirov** [comment]
Да, можно =)

Давно не занимался этим проектом, так как забросил область тестирования.
А при просмотре билдов в тимсити, отображаются ссылки на задачи? Это должно быть включено системно в тимсити.

**Comment by randomile** [comment]
Да, отображаются.  И ссылки активны, кликабельные, при наведении показывают доп. инфо о заявке.

Причем раньше работало корректно. Примерно до 27 апреля. Потом одновременно во всех билдах сломалось.
На стороне Teamcity, Jira, GitLab ничего не менялось существенного (версия например).

Что могло повлиять на работу?


С уважением, Виталий

**Comment by mamirov** [comment]
Плагин находится последним в иерархии шагов сборки?

**Comment by randomile** [comment]
да.
+ Создал отдельную сборку, где указан только этот шаг, ошибка та же


С уважением, Виталий

**Comment by mamirov** [comment]
К сожалению, я пока больше ничего не могу предположить.
Разве что могла поменяться версия TeamCity и в следствии чего, поменялась API.
Попробуйте посмотреть форки моего репозитория, там есть более новее версии, например эта:
https://github.com/jeffwysong/jirareporter/releases

Только там лежат исходники, придется самим собирать.

**Comment by randomile** [comment]
Спасибо.
Будем разбираться или искать альтернативы.

С уважением, Виталий

**Comment by Matsepura** [comment]
@randomile привет! нашли альтернативы?)

**Comment by randomile** [comment]
Привет.
В некотором смысле да.
Сделали форк и внесли корректировки под наши нужды :)
